docs/dev/agent_integration.md

```diff
@@ -6,35 +6,66 @@ aliases:
   - /developers/integrations/testing/
   - /integrations/datadog_checks_dev/
   - /guides/new_integration/
-
+  - /developers/integrations/new_check_howto/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/agent_integration.md"
+further_reading:
+- link: "https://www.datadoghq.com/blog/programmatically-manage-your-datadog-integrations/"
```

Comment:
Can we remove this blog from Further Reading? It's 5 years old and doesn't touch on Agent integrations. 

Could we instead add the "Create a Tile" page and whichever other resources we are pointing the user to in the main body of the documentation?

Reply:
@alai97 leaving this one up to you